<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>设备请求登录：</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>device_id:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leDeviceId"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>key:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="leDevicePassword"/>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="btnGetData">
           <property name="text">
            <string>设备登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3" rowspan="3">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>传感器操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1,0,1,0,1,0,1,0,1,0,1,0,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="btnGetCo2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>co2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGetUv">
               <property name="text">
                <string>紫外线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QPushButton" name="btnGetTe1">
               <property name="text">
                <string>温度1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGethu1">
               <property name="text">
                <string>湿度1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QPushButton" name="btnGetFire">
               <property name="text">
                <string>火光</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGetLight">
               <property name="text">
                <string>光照</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QPushButton" name="btnGetmq5">
               <property name="text">
                <string>甲烷</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGetMq2">
               <property name="text">
                <string>烟雾</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QPushButton" name="btnGetInf">
               <property name="text">
                <string>红外</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGetPm25">
               <property name="text">
                <string>pm2.5</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QPushButton" name="btnGetTe2">
               <property name="text">
                <string>温度2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnGetHu2">
               <property name="text">
                <string>湿度2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QPushButton" name="btnDS">
               <property name="text">
                <string>对射</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFS">
               <property name="text">
                <string>反射</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnUploadData">
             <property name="text">
              <string>数据上传</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>串口</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QComboBox" name="cbPortName"/>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="cbBaudRate">
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnOpenSerial">
             <property name="text">
              <string>打开串口</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>灯</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>高亮LED灯</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Light1:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="hsLight1">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Light2:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="hsLight2">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Light3:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="hsLight3">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Light4:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="hsLight4">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>炫彩LED灯</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnColor">
             <property name="text">
              <string>颜色</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2" rowspan="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>执行器操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>报警器：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="btnAlarmOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAlarmClose">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>报警灯：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="btnAlarmLedOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAlarmLedClose">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>舵机：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="btnSteerengineOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSteerengineClose">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>步进电机：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="btnStepmotorOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStepmotorClose">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>风扇：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="btnFanOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFanClose">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>继电器1：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QPushButton" name="btnRelay1Open">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRelay1Close">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>继电器2：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="btnRelay2Open">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRelay2Close">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>电磁锁：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QPushButton" name="btnLockOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnLockClose">
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextBrowser" name="tbShow"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnClearData">
        <property name="text">
         <string>清除窗口数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
